tests/drivers/spi/spi_loopback/boards/mimx8mm_phyboard_polis_mimx8mm6_m4.overlay

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2024 PHYTEC Messtechnik GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&ecspi1 {
+	status = "okay";
+	pinctrl-0 = <&ecspi1_default>;
+	pinctrl-names = "default";
```

Comment:
remove pinctrl-0 and pinctrl-names as have already in board dts?

Reply:
Done